loopchain/peer/block_manager.py

```diff
@@ -737,14 +737,14 @@ def __get_peer_stub_list(self):
 
         return max_height, unconfirmed_block_height, peer_stubs
 
-    def __close_level_db(self):
-        del self.__level_db
-        self.__level_db = None
-        self.__blockchain.close_blockchain_db()
+    def __close_key_value_store(self):
+        self.__key_value_store.close()
+        self.__key_value_store: KeyValueStore = None
+        self.__blockchain.close_blockchain_store()
 
     def stop(self):
-        # for reuse level db when restart channel.
-        self.__close_level_db()
+        # for reuse key value store when restart channel.
+        self.__close_key_value_store()
```

Comment:
Then  `BlockManager` don't need to close KeyValueStore.
Just call it `blockchain.close()`